project.clj

```diff
@@ -64,4 +65,4 @@
                         :compiler     {:main          com.github.discoverAI.snake.test-runner
                                        :output-to     "resources/public/js/compiled/tests.js"
                                        :output-dir    "resources/public/js/compiled/test/out"
-                                       :optimizations :none}}]})
+                                       :optimizations :none}}]})
```

Comment:
My editor removes EOL newlines, you can tell me if this fine or not.

Reply:
looks fine